content/applications/inventory_and_mrp/inventory/routes/strategies/storage_category.rst

```diff
@@ -0,0 +1,171 @@
+==================
+Storage categories
+==================
+
+A *storage category* is used with :doc:`putaway rules <putaway>`, as an extra location attribute to
+automatically propose optimal storage locations for products.
+
+Follow these steps to complete the setup:
+
+#. Enable the feature
+#. :ref:`Define storage category <inventory/routes/define-storage>` with specific limitations
+#. Assign :ref:`category to storage locations <inventory/routes/assign-location>`
+#. Add the storage category as an attribute to a :ref:`putaway rule
+   <inventory/routes/set-putaway-attribute>`
+
+.. seealso::
+   :doc:`putaway`
+
+.. note::
+   Assigning categories to storage locations tells Odoo that these locations that meet specific
+   requirements, such as temperature or accessibility. Odoo then evaluates these locations based on
+   defined capacity and recommends the best one on the warehouse transfer form.
+
+Configuration
+=============
+
+To enable storage categories, go to :menuselection:`Inventory app --> Configuration --> Settings`.
+First, in the :guilabel:`Warehouse` section, ensure the :guilabel:`Storage Locations` and
+:guilabel:`Multi-Step Routes` features are enabled.
+
+Then, activate the :guilabel:`Storage Categories` feature. Finally, click :guilabel:`Save`.
+
+.. image:: storage_category/enable-categories.png
+   :align: center
+   :alt: Show the Storage Categories feature.
+
+.. _inventory/routes/define-storage:
+
+Define storage category
+=======================
+
+A storage category with specific limitations must be created first before it is applied to locations
+to decide the optimal storage location.
+
+To create a storage category, go to :menuselection:`Inventory app --> Configuration --> Storage
+Categories` and click :guilabel:`Create`.
+
+On the storage category form, type a name for the :guilabel:`Storage Category` field.
+
+Options are available to limit capacity by weight, product, and package type.
+
+.. note::
+   Weight limits can be combined with capacity by package or product (e.g. a maximum of one hundred
+   products with a total weight of two hundred kilograms).
+
+   While it is possible to limit capacity by product and package type at the same location, it may
+   be more practical to store items in different amounts across various locations, as shown in this
+   :ref:`example <inventory/routes/set-capacity-package>`.
+
+The :guilabel:`Allow New Product` field defines when the location is considered available to store a
+product:
+
+- :guilabel:`If location is empty`: a product can be added there only if the location is empty.
+- :guilabel:`If products are the same`: a product can be added there only if the same product is
+  already there.
+- :guilabel:`Allow mixed products`: several different products can be stored in this location at
+  the same time.
+
+.. tip::
+   The :guilabel:`Location` smart button shows which storage locations the category has been
```

Comment:
```suggestion
   When clicked, the :guilabel:`Location` smart button shows which storage locations the category has been
```
adding the "when clicked..." preface to this sentence, because the Location smart button doesn't readily display such information, it needs to be clicked.